packages/postgres-database/src/simcore_postgres_database/models/api_keys.py

```diff
@@ -19,26 +19,50 @@
 api_keys = sa.Table(
     "api_keys",
     metadata,
-    sa.Column("id", sa.BigInteger(), nullable=False, primary_key=True),
-    sa.Column("display_name", sa.String(), nullable=False),
+    sa.Column(
+        "id",
+        sa.BigInteger(),
+        nullable=False,
+        primary_key=True,
+        doc="Primary key identifier",
+    ),
+    sa.Column(
+        "display_name",
+        sa.String(),
+        nullable=False,
+        doc="Human readable name. Unique for each user. SEE unique constraint below",
+    ),
     sa.Column(
         "user_id",
         sa.BigInteger(),
         sa.ForeignKey(users.c.id, ondelete="CASCADE"),
         nullable=False,
+        doc="This auth keys are indenfy this user",
+    ),
+    sa.Column(
+        "product_name",
+        sa.String,
+        sa.ForeignKey(
+            "products.name",
+            onupdate="CASCADE",
+            ondelete="CASCADE",
+            name="fk_api_keys_product_name",
+        ),
+        nullable=False,
+        doc="This auth keys are indenfy this product",
     ),
     sa.Column("api_key", sa.String(), nullable=False),
     sa.Column("api_secret", sa.String(), nullable=False),
     sa.Column(
         "created",
         sa.DateTime(),
-        nullable=False,
+        nullable=False,  # WARNING: still not updated to correct utc
         server_default=func.now(),
         doc="Timestamp auto-generated upon creation",
     ),
     sa.Column(
         "expires_at",
-        sa.DateTime(),
+        sa.DateTime(),  # WARNING: still not updated to correct utc
```

Comment:
yes but careful! My experience with utc  in the database is that it is a very delicate change and if not carefully done it can have a nasty impact. For that reason I do them in dedicated PRs